src/docs/integrations/slack/index.mdx

```diff
@@ -19,34 +19,31 @@ Declared in <code>config.yml</code>.
 
 <dd>
 <markdown>
-Copy your Client ID, Client Secret, and Verification Token.
+Copy your Client ID, Client Secret, and Signing Secret.
 
-Set `slack.legacy-app` to `False` if you made your bot Post June 2020, due to slack api changes
-
-If `slack.legacy-app` is `True` please use `slack.verification-token` instead of `slack.signing-secret`
 ```yaml
-slack.client-id: <'client id'>
+slack.client-id: <client id>
 slack.client-secret: <client secret>
 slack.signing-secret: <signing secret>
-slack.legacy-app: <boolean>
```

Comment:
We should update this file over at getsentry/onpremise too: https://github.com/getsentry/onpremise/blob/a1c0c1fd0ca8a4f3fef91a3129a81a125bdd48b5/sentry/config.example.yml#L103-L106